cmd/xl-v1-list-objects.go

```diff
@@ -38,6 +38,10 @@ func listDirFactory(ctx context.Context, disks ...StorageAPI) ListDirFunc {
 				continue
 			}
 
+			if len(entries) == 0 {
```

Comment:
Shouldn't we take care of quorum thing there ? I mean what if the first disk does not have the object here but all other have ?

Reply:
Quorum shall be validated at the top @vadmeste these entries get skipped invariably..

Reply:
> Quorum shall be validated at the top @vadmeste these entries get skipped invariably..

oh this is xl-v1 code, storage.Walk() is currently being used I guess